static/src/javascripts/projects/common/utils/template.js

```diff
@@ -1,15 +1,26 @@
+// jshint evil:true
+// jshint unused:false
+
 define([
-    'lodash/collections/reduce',
-    'lodash/objects/keys'
+    'lodash/objects/keys',
+    'common/views/svgs'
 ], function (
-    reduce,
-    keys
+    keys,
+    inlineSvg
 ) {
+    var svgRegEx = /{{(inlineSvg.*)?}}/g,
+        keyRegEx = /{{(.*?)}}/g;
 
-    return function template(tmpl, params) {
-        return reduce(keys(params), function (tmpl, token) {
-            return tmpl.replace(new RegExp('{{' + token + '}}', 'g'), params[token]);
-        }, tmpl);
-    };
+    // see docs/inline-svgs.md for info on how to insert an inline SVG in template
+    function svgReplacer(match, inlineSvgCall) {
+        return eval(inlineSvgCall);
```

Comment:
!

Not sure about this at all. Security implications aside, 'eval' stops the compiler / interpreter / whatever from being able to optimise things.